### DISCUSSION TITLE
How to optimize input parameters to achieve better accuracy for DPA2 than DP

### DISCUSSION THREAD

**Discussion by JiangXiaoMingSan** [deepmodeling/deepmd-kit#4651] [Q&A]
I trained DPA2 and DP potential functions using the same dataset, with accuracies of:
DPA2_Energy:7.1meV/atoms，Force：181meV/Å;DP_Energy:2.1meV/atoms，Force：117meV/Å
The two used identical datasets and similar hyperparameters, which was somewhat unexpected as I believe DPA-2's accuracy should be higher than DP's. Accuracy of DPA-2 during finetune stage：0.175meV/atoms，Force：43.7meV/Å.
The parameters I am currently using have undergone 2 iterations during the distillation stage, with an accuracy of iter000000:Energy:5.4meV/atoms，Force：200meV/Å,The accuracy improvement brought by iteration is very small, and the accuracy of energy even decreases,Can you give me some parameter suggestions for distillation to prevent such a significant decrease in accuracy.
Thank you for your attention to this issue, it is very important to me.
[train.json](https://github.com/user-attachments/files/19209621/train.json)
[distill_input.json](https://github.com/user-attachments/files/19210009/distill_input.json)

**Comment by njzjz** [comment]
The train.json file you post does not use DPA-2.

**Reply by JiangXiaoMingSan** [reply]
Thank you for your suggestion. I used the dpa2 descriptor for distillation and obtained a highly accurate potential function, with only a slight increase in error compared to finetune. However, the potential function trained by the dpa2 descriptor cannot be directly added to zbl and cannot be compressed, which makes the potential function almost unsuitable for simulation - it requires a particularly large amount of memory and is very slow. How should this be resolved, and will future updates focus on improving it?
